app/models/todo.rb

```diff
@@ -1,3 +1,6 @@
 class Todo < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :category

```

Comment:
We found that in Rails 5, a belongs_to association means that the associated record is required by default. FYI, in case you weren't aware of the change and the default behavior isn't what you wanted.

[http://blog.bigbinary.com/2016/02/15/rails-5-makes-belong-to-association-required-by-default.html](url)